packages/combobox/src/Combobox.ts

```diff
@@ -650,6 +661,7 @@ export class Combobox extends Textfield {
     public override disconnectedCallback(): void {
         this.itemObserver.disconnect();
         this.open = false;
+        this.resizeObserver?.unobserve(this);
```

Comment:
`unobserve` will only stop observing the specific `this`, if you later add more elements to observe they would still be observed and can lead to memory leaks if you forget to `unobserve` others. Rather lets do like this which is more robust and thorough.
```ts
 this.resizeObserver?.disconnect();
 this.resizeObserver = undefined;  // Clears the reference
```